src/AxeWindows.sln

```diff
@@ -24,6 +24,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Desktop", "Desktop\Desktop.
 	EndProjectSection
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DesktopTests", "DesktopTests\DesktopTests.csproj", "{AB9E59B6-54CE-4F5D-8055-8AD6745F3B84}"
+	ProjectSection(ProjectDependencies) = postProject
```

Comment:
Build order is InteropDummy, Desktop, then DesktopTests, but the analyzer can't figure that out the transitive dependency. This makes the dependency explicit to satisfy the analyzer, but has no impact on the actual build order.